builtin/providers/azurerm/resource_arm_app_service.go

```diff
@@ -0,0 +1,162 @@
+package azurerm
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/Azure/azure-sdk-for-go/arm/web"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceArmAppService() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceArmWebAppCreateUpdate,
+		Read:   resourceArmWebAppRead,
+		Update: resourceArmWebAppCreateUpdate,
+		Delete: resourceArmWebAppDelete,
+
+		Schema: map[string]*schema.Schema{
+			"resource_group_name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"skip_dns_registration": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  true,
+			},
+			"skip_custom_domain_verification": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  true,
+			},
+			"force_dns_registration": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  false,
+			},
+			"ttl_in_seconds": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Default:  "",
```

Comment:
ttl_in_seconds is passed in as a string

https://godoc.org/github.com/Azure/azure-sdk-for-go/arm/web#AppsClient.CreateOrUpdate